sample/build.gradle

```diff
@@ -122,3 +111,31 @@ android {
         }
     }
 }
+
+task syncPhotos(type: com.novoda.gradle.command.Files) {
+    deviceId {
+        def moto = android.command.devices().grep({ it.brand() == 'motorola' })
+        if (!moto) {
+            throw new GroovyRuntimeException('No Motorola device found')
+        }
+        moto[0].id
+    }
+    script {
+        def dir = '/sdcard/DCIM/Camera/'
+        def imageDir = file('motoPhoto')
```

Comment:
Acknowledged, this is a just a quick shot on file syncing. I'll update that.

Did you see that this only syncs files that are not already in the backup directory? That might deserve some explanation in the readme, may be.